proto/cosmos/group/v1beta1/tx.proto

```diff
@@ -309,3 +312,17 @@ message MsgExec {
 
 // MsgExecResponse is the Msg/Exec request type.
 message MsgExecResponse {}
+
+// MsgLeaveGroup is the Msg/LeaveGroup request type.
+message MsgLeaveGroup {
+  option (cosmos.msg.v1.signer) = "member_address";
+
+  // member_address is the account address of the group member.
+  string member_address = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
```

Comment:
maybe let's just use `address` (to be more consistent with `EventLeaveGroup` and `Member` types)?